tensorflow/core/platform/cuda_libdevice_path.h

```diff
@@ -36,7 +36,9 @@ inline std::vector<string> CandidateCudaRoots(
   }
 
   // "." is our last resort, even though it probably won't work.
-  candidates.push_back(".");
+  if (use_working_directory) {
+    candidates.push_back(".");
+  }
```

Comment:
I think this is a bug. `CandidateCudaRoots`'s `use_working_directory` argument would be unused otherwise.